package/libs/zlib/Makefile

```diff
@@ -48,6 +48,11 @@ endef
 
 TARGET_CFLAGS += $(FPIC)
 
+ifneq ($(findstring neon,$(CONFIG_TARGET_OPTIMIZATION)),)
+	CMAKE_OPTIONS += \
+		-DARMv8=ON
```

Comment:
Neon is also supported for ARMv7 CPUs.
Is it ok to activate the ARMv8 option on ARMv7 CPUs?

Reply:
Yes, it's safe.
https://github.com/madler/zlib/pull/251#issuecomment-326949213